app/components/clipboard_button_component.rb

```diff
@@ -11,4 +11,11 @@ def initialize(clipboard_text:, **tag_options)
   def call
     content_tag(:'lg-clipboard-button', super, data: { clipboard_text: clipboard_text })
   end
+
+  def content
+    safe_join [
+      render(IconComponent.new(icon: :content_copy, class: 'position-absolute')),
+      content_tag(:span, t('links.copy'), class: 'padding-left-3'),
+    ]
```

Comment:
supernit, doesn't matter, but it jumped out at me visually
(and unfortunately there doesn't seem to be a rubocop for this), but the rule I've learned and used for ruby and optional parens is statement vs expression

Since we're using the result of `safe_join` here (via implicit return) I think this should have parens

```suggestion
    safe_join([
      render(IconComponent.new(icon: :content_copy, class: 'position-absolute')),
      content_tag(:span, t('links.copy'), class: 'padding-left-3'),
    ])
```
and rubocop might make us do this one:
```suggestion
    safe_join(
      [
        render(IconComponent.new(icon: :content_copy, class: 'position-absolute')),
        content_tag(:span, t('links.copy'), class: 'padding-left-3'),
      ],
    )
```

Reply:
I had the first suggestion as the initial implementation, and Rubocop's complaints are what brought me to drop the parens, though I'm fine with the alternative as well (second suggestion).

Reply:
Updated in 91e695ed4.